app/build.gradle

```diff
@@ -0,0 +1,48 @@
+apply plugin: 'com.android.application'
```

Comment:
I noticed you renamed example/ to app/. Please rename back to example/ to minimize the changes in this PR